contrib/mmdarwin/mmdarwin.c

```diff
@@ -1,17 +1,10 @@
-/* Copyright 2019 Advens
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
-	* http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+/// \file     mmdarwin.c
+/// \authors  gcatto
+/// \version  1.0
+/// \date     08/11/18
+/// \license  GPLv3
```

Comment:
Is GPLv3 a hard requirement? rsyslog is (slowly but steadily) moving towards ASL 2.0 and being GPLv3 will prevent this module from ever become an official one.

Reply:
My mistake!

Reply:
thx a lot!